shardingsphere-test/shardingsphere-rewrite-test/src/test/resources/scenario/encrypt/case/alter_for_query_with_plain.xml

```diff
@@ -41,4 +46,6 @@
         <input sql="ALTER TABLE t_account_bak DROP COLUMN password , DROP COLUMN id" />
         <output sql="ALTER TABLE t_account_bak DROP COLUMN cipher_password  , DROP COLUMN assisted_query_password  , DROP COLUMN plain_password  , DROP COLUMN id" />
     </rewrite-assertion>
+
```

Comment:
Please remove redundant blank line.

Reply:
@strongduanmu please check, thanks!

Reply:
> @strongduanmu please check, thanks!

@arthasking123 Thank you for your efforts, but you should solve the check style problem that I mentioned before.